### ISSUE TITLE
[flowbroker-ui][remote node] The remote node is not added to the flowbroker-ui node list 

### ISSUE THREAD

**Issue by betefaber** [dojot/dojot#2400]
* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] support request

* **Do you want to request a *feature* or report a *bug*?**
bug

* **What is the current behavior?**
The remote node is not added to the flowbroker-ui node list

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**

- the remote node has been deployed

```
curl -X GET localhost:8000/flows/v1/node -H "Authorization: Bearer ${JWT}" -H "content-type: application/json" 2>/dev/null | jq
{
  "nodes": [
    {
      "id": "kelvin",
      "image": "dojot/kelvin-example:3.0.0-alpha2"
    }
  ]
}
```
- the node is added to the flowbroker node list

![Captura de tela de 2021-12-17 15-35-02](https://user-images.githubusercontent.com/37310063/146591935-6e6f9af5-4830-4422-b390-cfde5a02d05a.png)

- the node does not appear in the flowbroker-ui node list 

![image](https://user-images.githubusercontent.com/37310063/146591666-658cb848-21d6-4b2e-aa34-fd12a9e549da.png)

* **What is the expected behavior?**

* **What is the motivation / use case for changing the behavior?**

* **Please tell us about your environment:**
  
  - Version: flowbroker-ui/development
  - Environment: docker-compose
  - Operating system: Ubuntu 18.04

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

**Comment by cfrancisco** [comment]
That's correct. It is no longer possible to add remote nodes to Node-RED. :/
